src/patternfly/components/Spinner/spinner.scss

```diff
@@ -1,23 +1,34 @@
 // @debug $spinner; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
+:root,
 .#{$spinner} {
-  --#{$spinner}--diameter: var(--#{$pf-global}--icon--FontSize--xl);
+  --#{$spinner}--diameter: var(--pf-t--global--icon--size--2xl);
   --#{$spinner}--Width: var(--#{$spinner}--diameter);
   --#{$spinner}--Height: var(--#{$spinner}--diameter);
-  --#{$spinner}--Color: var(--#{$pf-global}--primary-color--100);
+  --#{$spinner}--Color: var(--pf-t--global--icon--color--brand--default);
```

Comment:
This seems like the right value but in figma it's defined as the border--color--brand token. cc @lboehling @andrew-ronaldson do you know which is right?